gapic-generator/README.md

```diff
@@ -41,6 +41,10 @@ build the generator from scratch.
 ```sh
 $ git clone https://github.com/googleapis/gapic-generator-ruby.git
 $ cd gapic-generator-ruby
+$ git submodule init
+$ git submodule update
+$ bundle install
+$ bundle exec rake
 $ gem build gapic-generator.gemspec
```

Comment:
Looks like this step is out of date since #18.

```sh
Chriss-iMac:gapic-generator-ruby quartzmo$ gem build gapic-generator.gemspec
ERROR:  Gemspec file not found: gapic-generator.gemspec
```

Reply:
Fixed!